### ISSUE TITLE
Indiafoss 2026 landing page accessibility issue

### ISSUE THREAD

**Issue by idlip** [fossunited/fossunited#1669]
- Timeline part seems to not be announced completely well, the dates and name should synced well together to be read
  View is fine, but for Audio we need to say it as viewers read in their mind

eg: schedule publish on 01 September - will be nice

- Actions card does not have Interactive semantics:
  https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Keyboard#Focusable_elements_should_have_interactive_semantics

  Also pre-events is not announced for me

- Sponsors must be announced by tiers and not generic as "premier sponsors"
   Use either Maintainer or Contributor tier properly

- Tabbing for links should start from viewport and focus region (not from top all time)

- Make FAQ as last tab priority, since we need people to notice child care and accessibility form early on
